husky_components/overlay/main.js

```diff
@@ -702,9 +707,10 @@ define([], function() {
 
             this.sandbox.dom.addClass(this.overlay.$el, 'has-language-chooser');
             this.overlay.slides[slide].$languageChanger = this.sandbox.dom.createElement(
-                '<div class="' + constants.languageChangerClass + '"/>'
+                this.slides[slide].tabs !== null?'<div class="' + constants.languageChangerClass + '"/>' :
+                '<div style="height: 51px;"/> <div class="' + constants.languageChangerClass + '-frame"/>'
```

Comment:
Inline styles are generally considered bad practice. Try to use another mechanism to set your styles. Preferably with CSS.